packages/client/components/ReflectionGroup/DraggableReflectionCard.tsx

```diff
@@ -67,9 +68,10 @@ const DraggableReflectionCard = (props: Props) => {
   const {
     isClipped,
     reflection,
-    staticIdx,
+    staticIdx = 0,
```

Comment:
Dragging the reflection when it's in the Spotlight doesn't work yet - it clones the reflection. I think that's out of scope here and plan on tackling that in the issue where we need to drag the Spotlight reflection to the reflection groups. 

To resolve that, I plan on digging into `useDraggableReflectionCard` in more detail and seeing how `staticIdx` and `staticReflectionCount` are used